resources/clients/react/elwing/src/plugins/item-tracker/work-item/WorkItemListComponent.tsx

```diff
@@ -0,0 +1,179 @@
+/**
+ * An element that displays a list of work items that are retrieved from a REST service.
+ *
+ * - Select Active or Archived to display work items with the specified state.
+ * - Select the wastebasket icon to archive an active item.
+ * - Select 'Add item' to add a new item.
+ * - Enter a recipient email and select 'Send report' to send a report of work items.
+ */
+
+import { useEffect, useState } from "react";
+import { WorkItem, WorkItemStatus } from "./WorkItemService";
+import { workItemService as service } from "./WorkItemService";
+import {
+  Alert,
+  Button,
+  FormField,
+  Input,
+  Select,
+  SpaceBetween,
+  Table,
+} from "@cloudscape-design/components";
+import { OptionDefinition } from "@cloudscape-design/components/internal/components/option/interfaces";
+import { useItemTrackerAction, useItemTrackerState } from "../ItemTrackerStore";
+
+const STATUS_OPTIONS: OptionDefinition[] = [
```

Comment:
Would an object be more clear here?
e.g.
```javascript
const StatusOptions = {
  Active: { value: "active", label: "Active" }
}
```

Reply:
I don't think so? This is 1:1 w/ the Cloudscape API. (Unless I missed this as an alternative in their docs)